_data/blog_authors.yml

```diff
@@ -33,3 +33,8 @@ meysam:
     role: Staff AI Researcher
     linkedin: https://www.linkedin.com/in/meysam-ac/
     picture: /assets/img/faces/faces_meysam.jpg
+damian:
+    name: Damian Bojar
+    role: Backend Chapter Lead
+    linkedin: https://www.linkedin.com/in/bojar-damian
+    picture: /assets/img/faces/faces_damian.jpg
```

Comment:
not sure if the picture would be found like this? as the ending is jpeg
Have you tried???

Reply:
Just checked, it will not be found like that